app/models/attachment.rb

```diff
@@ -25,4 +26,9 @@ def update_attributes(new_params)
   def changed?
     @changed
   end
+
+  def self.all_from_publishing_api(payload)
```

Comment:
I'm not 100% sure about the name for this method. Initially when I saw it, I thought it would be similar to `Manual.all`, not extracting the attachments from the Publishing API payload.

What do others think?